Sammuel09/Fast-Food-Fast
# #160704302 User Create order endpoint (user)

#### What does this PR do?
Allows a user create an order 

#### Description of Task to be completed?
- Users can make orders
- The route is authenticated such that users need to be logged in or signed up to access the route.

### How should this be manually tested?
First, signup to the app by making a POST request to https://sammie-fast-food-fast.herokuapp.com/api/v1/orders

Use data in the following format:
`username: string`,
`email: string`,
`phone: string`,
`address: string`,
`password: string`

Or signin with the following user credentials at https://sammie-fast-food-fast.herokuapp.com/api/v1/auth/login

`email: tobi@gmail.com`
`password: tobi`

Copy the token received in the response in the signup process and add it as part of the request headers in this format:
`x-access-token: token`

On Postman, visit https://sammie-fast-food-fast.herokuapp.com/api/v1/menu
Then make a POST Request with JSON Data in the following format:
`userId: number`,
`menuId: number`,
`quantity: 2`,
`instruction: string`

NB: Instruction represents delivery instruction.

Expected Data to be returned:
`status: string`,
`data: object`,
`message: string`

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/n/projects/2196965/stories/160704302
